src/Home/Home.spec.tsx

```diff
@@ -355,16 +386,76 @@ describe("Home", () => {
           const alert = jest.spyOn(Alert, "alert")
 
           fireEvent.press(fixProximityTracingButton)
-          await waitFor(() => {
-            expect(alert).toHaveBeenCalledWith(
-              "Authorize in Settings",
-              "To activate Proximity Tracing, authorize COVID-19 Exposure Logging in the Settings app",
-              [
-                expect.objectContaining({ text: "Back" }),
-                expect.objectContaining({ text: "Open Settings" }),
-              ],
-            )
-          })
+          expect(alert).toHaveBeenCalledWith(
+            "Authorize in Settings",
+            "To activate Proximity Tracing, authorize COVID-19 Exposure Logging in the Settings app",
+            [
+              expect.objectContaining({ text: "Back" }),
+              expect.objectContaining({ text: "Open Settings" }),
+            ],
+          )
+        })
+      })
+    })
+    describe("when the device requires location permissions", () => {
```

Comment:
```suggestion
    describe("when the device platform is android", () => {
      it("shows the location activation status", () => {
```

Reply:
Is this true? I thought this was just for certain versions of Android? Or am I mistaken?

Reply:
Good question! I am not sure, please only follow through on suggestions if they make sense. I do know that it will never be required on iOS. If its only a subset of android devices require it, are we showing location status on all android devices? should we?

Reply:
We are only showing location status on Android devices if the device does not support locationless scanning.

Reply:
I'm thinking this:

```
describe("when the device does not support locationless scanning", () => {
```